chatty-clownfish/bangazon-sprint-2
# Dw 1/1 tick18 view profile

## Link to Ticket
Closes #50

## Description of Proposed Changes
If the user is authenticated/logged in, they can click on "My Profile" in the nav bar. They will then be taken to the profile view (profile/list.html). Here they can view their profile information (first name, last name is coming from the user table. And address, phone number is coming from the customer table). They will also see their payment options (coming from the paymenttypes table).
- 

## Steps to Test

- Login as any user

- Click "My Profile" in the nav bar

- Make sure that the displayed information matches the users data in the database

```sh
git fetch --all
git checkout dw-1/1_tick18_viewProfile
```


## Impacted Areas in Application

List general components of the application that this PR will affect:

* Profile
